src/server/pps/server/s3g_sidecar.go

```diff
@@ -193,69 +207,16 @@ func (s *s3InstanceCreatingJobHandler) OnCreate(ctx context.Context, jobInfo *pp
 	// more than one job in s.servers). When parallel jobs are implemented, the
 	// servers in s.servers won't actually serve anymore, and instead parent
 	// server will forward requests based on the request hostname
-	port := s.s.apiServer.env.Config().S3GatewayPort
```

Comment:
^ The TODO above can probably go away now?

Reply:
ah yes!